autobuild.xml

```diff
@@ -584,11 +614,11 @@
             <key>archive</key>
             <map>
               <key>hash</key>
-              <string>4cd82e2dec06ddff19e9b3dc0254f2593ec80452</string>
+              <string>0cafca1370b272e7aa717f239c90750b0c19e307</string>
               <key>hash_algorithm</key>
               <string>sha1</string>
               <key>url</key>
-              <string>https://github.com/secondlife/3p-expat/releases/download/v2.1.1.1f36d02/expat-2.1.1.1f36d02-linux64-1f36d02.tar.zst</string>
+              <string>https://github.com/secondlife/3p-expat/releases/download/v2.6.2-r3/expat-2.6.2-r3-linux64-8583963182.tar.zst</string>
```

Comment:
It looks like you're updating expat, but only on linux. We generally try to update all platforms at once, and I have been working on that here: https://github.com/secondlife/viewer/issues/1078 Could you remove the autobuild.xml changes that are being worked on under #1078?